devenv/README.md

```diff
@@ -2,15 +2,30 @@ This folder contains useful scripts and configuration for...
 
 * Configuring dev datasources in Grafana
 * Configuring dev & test scenarios dashboards.
+* Creating docker-compose file with DBs and fake data.
+
+
+# Dev dashboards and data sources
 
 ```bash
 ./setup.sh
 ```
 
-After restarting grafana server there should now be a number of datasources named `gdev-<type>` provisioned as well as a dashboard folder named `gdev dashboards`. This folder contains dashboard & panel features tests dashboards. 
+After restarting grafana server there should now be a number of datasources named `gdev-<type>` provisioned as well as
+a dashboard folder named `gdev dashboards`. This folder contains dashboard & panel features tests dashboards. 
+
+#### Dev dashboards
 
-# Dev dashboards
+Please update these dashboards or make new ones as new panels & dashboards features are developed or new bugs are
+found. The dashboards are located in the `devenv/dev-dashboards` folder. 
 
-Please update these dashboards or make new ones as new panels & dashboards features are developed or new bugs are found. The dashboards are located in the `devenv/dev-dashboards` folder. 
+# docker-compose with databases
+
+```bash
+./create_docker_compose.sh [...docker_block_name]
```

Comment:
Would be great with a concrete example here as well:

```bash
./create_docker_compose.sh influxdb prometheus2 elastic5
```

Reply:
Yes that is probably better, will change.